docs/tutorials/gke.md

```diff
@@ -14,7 +14,7 @@ This solution will only work when both CloudDNS and GKE are provisioned in the s
 
 ### Configure Project Environment
 
-Setup your environment to work with Google Cloud Platform. Fill in your variables as needed, e.g. target project.
+Set up your environment to work with Google Cloud Platform. Fill in your variables as needed, e.g. target project.
```

Comment:
the word `set up` is used to [make  arrangements](https://www.yourdictionary.com/articles/setup-set-up-differences)